### DISCUSSION TITLE
so-elastalert ERROR

### DISCUSSION THREAD

**Discussion by SafelineMan** [Security-Onion-Solutions/securityonion#2447] [Unsupported Versions] [answered]
so-elastalert Always in error state， so-elastalert and sudo docker restart $(sudo docker ps -a -q)  cant solve it，What should I do next to make it work
 
![image](https://user-images.githubusercontent.com/33274442/103065461-9558ed00-45f1-11eb-8bf0-917fd82b61bc.png)

**Comment by SafelineMan** [comment]
who can help me

**Comment by TOoSmOotH** [comment]
Well I guess maybe the first place to look would be the log in /opt/so/log/elastalert

**Reply by SafelineMan** [reply]
it seems to have been out of work for many days

![image](https://user-images.githubusercontent.com/33274442/103184822-aa38c780-48f4-11eb-8e4f-822a8be4257d.png)

**Comment by defensivedepth** [comment]
Do you get any errors when you run `sudo so-elastalert-restart --force` ?

**Reply by SafelineMan** [reply]
When I run sudo so-elastalert-restart --force, there is no error。but sudo so-status its still error

![image](https://user-images.githubusercontent.com/33274442/103253101-bd15cf80-49ba-11eb-8486-086da4080a9d.png)

**Reply by defensivedepth** [reply]
Tail the log right after restarting elastalert, you should see startup logs:

`sudo so-elastalert-restart --force && sudo tail -f /opt/so/log/elastalert/elastalert.log`

**Reply by SafelineMan** [reply]
emmmm..... startup logs see below, I closed all the tasks in the script，what can i do next  thank you
![image](https://user-images.githubusercontent.com/33274442/103254022-aa9d9500-49be-11eb-954b-a40feaabc7af.png)

**Comment by SafelineMan** [comment]
i need help please~~

**Reply by SafelineMan** [reply]
help   ~~~~~~~~~~~~~~~~~~~~

**Comment by jertel** [comment, marked as answer]
What does `docker logs so-elastalert` tell you?

**Reply by SafelineMan** [reply]
thank you very much ,   i found it is a playbook error , when i remove it   , it  start succss.


![image](https://user-images.githubusercontent.com/33274442/103603825-7724b280-4f4a-11eb-9107-40bcc8430243.png)

![image](https://user-images.githubusercontent.com/33274442/103603846-8441a180-4f4a-11eb-92cb-dcac1e98ed43.png)
